include/net/nrf_cloud.h

```diff
@@ -239,6 +239,19 @@ struct nrf_cloud_init_param {
  */
 int nrf_cloud_init(const struct nrf_cloud_init_param *param);
 
+/**
+ * @brief Uninitialize nRF Cloud; disconnects cloud connection
+ *  and cleans up allocated memory.
+ *
+ * @note If nRF Cloud FOTA is enabled and a FOTA job is active
+ *  uninit will not be performed.
+ *
+ * @retval 0      If successful.
+ *         -EBUSY If a FOTA job is in progress.
```

Comment:
AFAIK each possible return value should have a separate `@retval` tag.

Reply:
i did not know this, thanks!